frontend/src/components/CognitoLoginAndReg.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import { useLocation } from 'react-router-dom'
+
+function useQuery() {
+  return new URLSearchParams(useLocation().search);
+}
+
+export default () => {
```

Comment:
This component is called `CognitoLoginAndReg`, yet it seems to contain only login-related code. Is it because it's not finished, or because it's misnamed?

Reply:
The URL which the login button redirects to opens a UI provided by Cognito. That UI handles both login, registration and password recovery.